handel/src/cli/index.ts

```diff
@@ -321,16 +330,14 @@ export async function checkAction(handelFile: HandelFile, argv: any): Promise<vo
  * This method is the top-level entry point for the 'delete' action available in the
  * Handel CLI. It asks for a confirmation, then deletes the requested environment.
  */
-export async function deleteAction(handelFile: HandelFile, argv: any): Promise<void> {
-    configureLogger(argv);
+export async function deleteAction(handelFile: HandelFile, options: DeleteOptions): Promise<void> {
+    configureLogger(options);
 
-    const environmentToDelete = argv.e;
     try {
         await validateLoggedIn();
-        const accountConfig = await config(argv.c); // Load account config to be consumed by the library
+        const accountConfig = await config(options.accountConfig); // Load account config to be consumed by the library
         await validateCredentials(accountConfig);
-        const deleteEnvConfirmed = await confirmDelete(environmentToDelete, argv.y);
-        if (deleteEnvConfirmed) {
+        for (const environmentToDelete of options.environments) {
```

Comment:
Turns out that, even though it theoretically supported multiple environments, handel delete didn't.

Reply:
Yes I started out supporting multiple environment deletes intentionally, but then rolled back to only allowing a single one at a time. The reasoning for this was that delete is such a dangerous operation (deleting dbs, buckets, file mounts, etc.) that you should have to do the environment deletes one at a time rather than being able to delete multiples in bulk.

What do you think about that reasoning? I'm inclined to be overly cautious here personally, but maybe I'm just being too paranoid.

Reply:
@dsw88 done